src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class Figure implements Draw {
+
+    protected Color color;
+
+    protected Figure(Color color) {
+        this.color = color;
+    }
+
+    public abstract double getArea();
+
+    @Override
+    public void draw() {
+        System.out.print(String.format("%s. Color: %s. ", this.getClass().getSimpleName(), color));
```

Comment:
Use name() for getting String representation of enum constants. Instead of using color directly, you should use color.name() to ensure the proper String representation is used.